src/services/ActivityCardService.js

```diff
@@ -1 +1,146 @@
-// placeholder file for Drive shenanigans woohoo!
+import {
+  checkFileExists,
+  writeFile,
+  makeDirectory,
+  moveFile,
+  deleteFile,
+} from './routes/Local';
+import { MAINDIRECTORY } from './constants';
+import { AccessToken } from './models';
+import axios from 'axios';
+import { DRIVE_API_URLS } from '../config.json';
+
+const ActivityCardService = {
+  // All APIs for ActivityCards should be here
+  // Don't call RNFS directly here!
+  // Instead, call a function from Local.js :~]
+
+  // const auth = await google.auth.getClient({
+  //   credentials: YOUR_AUTH_CREDENTIALS,
+  //   scopes: ['https://www.googleapis.com/auth/drive'],
+  // });
+
+  /**
+   * Returns array of file paths for this month's featured activity cards.
+   * @return {String[]}
+   */
+  getFeaturedActivityCards: async function () {
+    try {
+      //Retrieve current year and month from browser
+      const today = new Date();
+      var currYear = String(today.getFullYear());
+      var currMonth = String(today.getMonth() + 1).padStart(2, '0'); //Account for January being '0'
+
+      //Set up GET url and path to Featured Card directory
+      var downloadUrl =
+        DRIVE_API_URLS.SEARCH_FILES +
+        'supportsAllDrives=true&trashed=false&includeItemsFromAllDrives=true&fields=files(id,kind,driveId,mimeType,teamDriveId,name,createdTime)';
+      var path = MAINDIRECTORY + '/FeaturedActivityCards/';
+      let driveFiles = {};
+
+      //Delete anything that may currently be in the Featured Cards directory, make the new path with no contents
+      if (await checkFileExists(path)) {
+        await deleteFile(path);
+      }
+      await makeDirectory(path);
+
+      //Retreive all Drive files
+      await axios
+        .get(downloadUrl)
+        .then(response => {
+          driveFiles = response.data;
+        })
+        .catch(error => {
+          console.error('ERROR OCCURRED: ' + error);
```

Comment:
```suggestion
          console.error('getFeaturedActivityCards: ' + error);
```
Better when reading through logs to know which endpoint caused what error